lib/pbio/platform/technic_hub/startup.s

```diff
@@ -76,6 +76,7 @@ defined in linker script */
 	.weak	Reset_Handler
 	.type	Reset_Handler, %function
 Reset_Handler:
+  cpsid i   /* Disable interrupts in case LEGO bootloader left them enabled before handover. */
```

Comment:
nit: we know for certain that it leaves them enabled (from reverse engineering), so saying "in case" seems a bit misleading

Reply:
yes